### ISSUE TITLE
Hooks with "private" tasks

### ISSUE THREAD

**Issue by mgzenitech** [nat-n/poethepoet#133]
How can I setup the hook without exposing the task in the list?

```toml
[tool.poe.tasks._aaa]
shell = "echo aaaaaaaaaaaaaaaaaaaaaaaaaaaa"

[tool.poe.poetry_hooks]
post_install = "_aaa"
```

This triggers the following error:
```
Error: Tasks prefixed with `_` cannot be executed directly 
```

**Comment by nat-n** [comment]
Hi @mgzenitech, thanks for reporting this. Looks like a bug, I'll see what I can do about it.

**Comment by nat-n** [comment]
It's fixed in [0.19.0](https://pypi.org/project/poethepoet/0.19.0/) 🙂
